client/views/create/create.js

```diff
@@ -90,6 +90,9 @@ Template.create.events({
     const maxResponse = responseSelect[responseSelect.selectedIndex].value;
     const hiddenSelector = document.getElementsByName('visibility')[0];
     const isHidden = (hiddenSelector[hiddenSelector.selectedIndex].value === 'hidden');
+    const socialSelector = document.getElementsByName("social")[0]; // Extract the HTML select element for getting the social toggles
```

Comment:
@singhsanket143 can you use the single quote as in the `hiddenSelector` var?

Reply:
Updated the quotes here.